falcon/util/misc.py

```diff
@@ -210,3 +214,33 @@ def get_bound_method(obj, method_name):
             raise AttributeError(msg)
 
     return method
+
+
+def get_http_status(status_code):
+    """Gets both the http status code and description from just a code
+
+    Args:
+        status_code: integer or string that can be converted to an integer
+
+    Returns:
+        str: status code e.g. "404 Not Found"
+
+    Raises:
+        ValueError: the value entered could not be converted to an integer
+
+    """
+    # sanitize inputs
+    try:
+        code = decimal.Decimal(str(status_code))  # Decimal can validate values like "401.1"
+        code = int(code)  # Converting to int removes the decimal places
+        if code < 100:
+            raise ValueError
+    except (decimal.InvalidOperation, ValueError):
+        raise ValueError('get_http_status "%s" is not a valid status code', status_code)
+
+    # lookup the status code
+    try:
+        return getattr(status_codes, 'HTTP_' + str(code))
+    except AttributeError:
+        # not found
+        return str(code) + ' Unknown Status Code'
```

Comment:
That makes sense, but if it is intentional, wouldn't the caller want to supply their own description string?